extractor/src/main/scala/com/digitalasset/extractor/helpers/TransactionTreeTrimmer.scala

```diff
@@ -26,6 +30,15 @@ object TransactionTreeTrimmer {
     case Kind.Empty => false
   }
 
+  private def exerciseEventOrStakeholder(parties: Set[String]): TreeEvent.Kind => Boolean = {
+    case Kind.Created(event) =>
+      event.signatories.toSet.intersect(parties).nonEmpty || event.observers.toSet
```

Comment:
```suggestion
      event.signatories.exists(parties) || event.observers
```

(and similarly for `observers`)